src/test_driver/openiotsdk/integration-tests/shell/test_app.py

```diff
@@ -1,17 +1,19 @@
-# Copyright (c) 2009-2021 Arm Limited
-# SPDX-License-Identifier: Apache-2.0
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
```

Comment:
Why do we change the copyright notice ?

Reply:
To set the right date and unify all copyright notes in integration tests.